app/workers/feeds/import_articles_worker.rb

```diff
@@ -18,7 +19,21 @@ def perform(user_ids = [], earlier_than = nil)
         earlier_than ||= 4.hours.ago
       end
 
-      ::Feeds::Import.call(users_scope: users_scope, earlier_than: earlier_than)
+      users_scope.select(:id).find_in_batches do |batch|
+        ids = batch.map { |user| [user.id] }
+
+        ForUser.perform_bulk ids, earlier_than
```

Comment:
nit: minor styling consistency detail. Doesn't our husky hook fix this?

```suggestion
        ForUser.perform_bulk(ids, earlier_than)
```

Reply:
Fixed in https://github.com/forem/forem/pull/16818/commits/9e9634a2174cea828dad2c5bf80a9e66ea00f1d1

I didn't realize we actually did this so consistently. I could've sworn I saw more places where we didn't use parens. To be quite honest, I don't love them for command methods, but I think they make perfect sense for methods that we're using for their return values. Either way, I concede to consistency here.